### PR TITLE
frontend: migrated tests from enzyme to RTL

### PR DESCRIPTION
**Pull request by shonsirsha**
As enzyme doesn’t have support for the latest React versions, we would like to migrate our tests from enzyme to
react-testing-library (RTL).

### CODE REVIEW

File: frontends/web/src/utils/markup.test.tsx [hunk 1]
```diff
@@ -15,38 +15,39 @@
...
 import { multilineMarkup, SimpleMarkup } from './markup';
 
 describe('SimpleMarkup', () => {
   it('contains a strong element with the text bar', () => {
-    const paragaph = mount(<SimpleMarkup tagName="p" markup="foo <strong>bar</strong> baz" />);
-    expect(paragaph.getDOMNode().nodeName).toEqual('P');
-    expect(paragaph.find('strong').text()).toEqual('bar');
+    const { container } = render(<SimpleMarkup tagName="p" markup="foo <strong>bar</strong> baz" />);
+    // container is a div that wraps whatever element that's being rendered
+    expect(container.firstElementChild?.nodeName).toBe('P');
```
Comment (shonsirsha) [inline frontends/web/src/utils/markup.test.tsx:25]: Could avoid having `firstElementChild` by using `data-testid` for the tested element (SimpleMarkup). 

But, with the way SimpleMarkup works rn, adding `data-testid` might only increase the code's complexity without adding real value / benefits.

**Comment by shonsirsha** [comment]
As `enzyme` is not being used anywhere anymore, could we safely remove packages that are related to it? If so, ill remove it together in this PR.

Related packages:

1. `enzyme`
2. `enzyme-to-json`
3. `@types/enzyme`
4. `@wojtekmaj/enzyme-adapter-react-17`

**Comment by shonsirsha** [comment]
Because `enzyme` is not being used anywhere anymore, could we safely remove packages that are related to it? If so, ill remove them together in this PR.

Related packages:

1. `enzyme`
2. `enzyme-to-json`
3. `@types/enzyme`
4. `@wojtekmaj/enzyme-adapter-react-17`


please lmk, thanks 🙂

**Comment by thisconnect** [comment]
@shonsirsha very nice, yes please remove the packages in this PR 👍

**Comment by shonsirsha** [comment]
@thisconnect Great! Enzyme-related packages have been removed from the app.

**Review by thisconnect — APPROVED** [review]
tested LGTM 👍

**Pull request merged**